### ISSUE TITLE
Could Timeline data items be connected as in Network Nodes?

### ISSUE THREAD

**Issue by graemetshaw** [almende/vis#1515]
I have been given a requirement that the data items in my timeline have a sequential dependency.

e.g. Item 23(start: x, end: y) must complete before Item 42(start:a, end: b) can begin, otherwise Item 42 must be rescheduled.

We would like to visualise this by connecting item 23(end) with Item 42(start).

Is this possible or planned? Or can this be done by adding an EdgeHandler to manage the connection between the Items?

Your advice would be appreciated.

**Comment by josdejong** [comment]
Sounds like you're looking for a Gantt chart or something. This is not supported nor planned by ourselves. vis.js is an open source project though, contributions are welcome :).

**Comment by yotamberk** [comment]
Duplicate of #1515

**Comment by mojoaxel** [comment]
duplicate of #1699
